pkg/virtctl/create/instancetype/instancetype.go

```diff
@@ -107,16 +113,27 @@ func NewCommand() *cobra.Command {
 	return cmd
 }
 
-func (c *createInstancetype) setDefaults(cmd *cobra.Command) {
+func (c *createInstancetype) setDefaults(cmd *cobra.Command) error {
+	namespace, overridden, err := c.clientConfig.Namespace()
+	if err != nil {
+		return err
+	}
+	if overridden {
+		c.namespace = namespace
+		c.namespaced = true
```

Comment:
Do we still need the `namespaced` flag? We could by default create `VirtualMachineClusterInstancetype` and based on `namespace` flag decide if we want the `VirtualMachineInstancetype` instead. What do you think about that?

Reply:
I'd keep the `namespaced` flag so it is still possible to create a `VirtualMachineInstancetype` that does not have the namespace set.